pkg/util/mount/mount_windows.go

```diff
@@ -209,10 +209,9 @@ func getDeviceNameFromMount(mounter Interface, mountPath, pluginDir string) (str
 	if len(refs) == 0 {
 		return "", fmt.Errorf("directory %s is not mounted", mountPath)
 	}
-	basemountPath := normalizeWindowsPath(path.Join(pluginDir, MountsInGlobalPDPath))
 	for _, ref := range refs {
-		if strings.Contains(ref, basemountPath) {
-			volumeID, err := filepath.Rel(normalizeWindowsPath(basemountPath), ref)
+		if strings.Contains(ref, pluginDir) {
```

Comment:
does plugindir need to be normalized first?

Reply:
Yes, it does. Good catch.